src/coreclr/src/jit/codegencommon.cpp

```diff
@@ -5478,13 +5456,8 @@ void CodeGen::genFreeLclFrame(unsigned frameSize, /* IN OUT */ bool* pUnwindStar
     }
     else
     {
-        regMaskTP grabMask = RBM_INT_CALLEE_TRASH;
-        if (jmpEpilog)
-        {
-            // Do not use argument registers as scratch registers in the jmp epilog.
-            grabMask &= ~genJmpCallArgMask();
-        }
-        regNumber tmpReg = REG_TMP_0;
```

Comment:
The initial version of that code (6+ years) had a comment:
```
        // JBToDo: Allocate a register for this, rather than assuming r3 isn't used in
        // the epilog
```
but that was lost.